src/Requests.jl

```diff
@@ -14,6 +14,10 @@ else
     import Base.Filesystem: File
 end
 
+if VERSION >= v"0.5.0-dev+4817"
+    import Base: view
+end
+
```

Comment:
There is this in Compat:
````
if !isdefined(Base, :view)
    const view = slice
end
````
But isn't import is still required, since Requests also defines `view`?